umbracocms-webapp-with-storage-and-cache/azuredeploy.json

```diff
@@ -0,0 +1,576 @@
+         ],
+         "metadata":{
+            "description":"Redis Cache Capacity  , supported values from 0 to 6"
+         },
+         "defaultValue":0
+      },
+      "redisCacheVersion":{
+         "type":"string",
+         "defaultValue":"2.8",
+         "metadata":{
+            "description":"Redis Cache version 2.8"
+         }
+      },
+      "NonAdmin_DatabaseUsername":{
+         "type":"string",
+         "metadata":{
+            "description":"Non-admin Database User. Must be Unique"
+         }
+      },
+      "NonAdmin_DatabasePassword":{
+         "type":"securestring",
+         "metadata":{
+            "description":"Non-admin Database User password"
+         }
+      }
+   },
+   "resources":[
+      {
+         "apiVersion":"2014-04-01-preview",
+         "name":"[parameters('redisCacheName')]",
+         "type":"Microsoft.Cache/Redis",
+         "location":"[parameters('siteLocation')]",
+         "properties":{
+            "sku":{
+               "name":"[parameters('redisCacheSKU')]",
+               "family":"[parameters('redisCacheFamily')]",
+               "capacity":"[parameters('redisCacheCapacity')]"
+            },
+            "redisVersion":"[parameters('redisCacheVersion')]",
+            "enableNonSslPort":true
+         }
+      },
+      {
+         "apiVersion":"2.0",
+         "name":"[parameters('serverName')]",
+         "type":"Microsoft.Sql/servers",
+         "location":"[parameters('serverLocation')]",
+         "properties":{
+            "administratorLogin":"[parameters('administratorLogin')]",
+            "administratorLoginPassword":"[parameters('administratorLoginPassword')]"
+         },
+         "resources":[
+            {
+               "apiVersion":"2.0",
+               "name":"[parameters('databaseName')]",
+               "type":"databases",
+               "location":"[parameters('serverLocation')]",
+               "dependsOn":[
+                  "[concat('Microsoft.Sql/servers/', parameters('serverName'))]"
+               ],
+               "properties":{
+                  "edition":"[parameters('edition')]",
+                  "collation":"[parameters('collation')]",
+                  "maxSizeBytes":"[parameters('maxSizeBytes')]",
+                  "requestedServiceObjectiveId":"[parameters('requestedServiceObjectiveId')]"
+               }
+            },
+            {
+               "apiVersion":"2.0",
+               "name":"AllowAllWindowsAzureIps",
+               "type":"firewallrules",
+               "location":"[parameters('serverLocation')]",
+               "dependsOn":[
+                  "[concat('Microsoft.Sql/servers/', parameters('serverName'))]"
+               ],
+               "properties":{
+                  "endIpAddress":"0.0.0.0",
+                  "startIpAddress":"0.0.0.0"
+               }
+            }
+         ]
+      },
+      {
+         "apiVersion":"2014-06-01",
+         "type":"Microsoft.ClassicStorage/storageAccounts",
```

Comment:
This should be a v2 storage account. Only portal and certain other subs can create classic resources through ARM